neqo-transport/src/connection/tests/stream.rs

```diff
@@ -582,6 +617,31 @@ fn simultaneous_stop_sending_and_reset() {
     );
 }
 
+#[test]
+//The client sends a duplicate stream reset.
```

Comment:
```suggestion
//The client sends a duplicate stream reset.
#[test]
```

Seems to be the convention in the codebase.

See also https://github.com/rust-lang/style-team/issues/72. I couldn't find a clippy lint.

Reply:
I use doc comments for tests.

Which would make this `///` instead.